runner/operator/access/v1_0_0/snps_and_indels/test/fixtures/curated_normals_file_group.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+      "model": "file_system.filegroup",
+      "pk": "a66f03b4-ce43-40d0-9134-d5a3575cabcc",
+      "fields": {
+          "created_date": "2019-11-05T20:13:29.923Z",
+          "modified_date": "2019-11-05T20:13:29.923Z",
+          "name": "access_curated_normals",
+          "slug": "access_curated_normals",
+          "storage": "0cd14133-0f6a-413d-a498-495bf48546ee"
```

Comment:
Try linking to the common file storage fixture here, then we shouldn't need to add a new fixture for file storage